### PR TITLE
✨Data seeder+other bits

### PR DESCRIPTION
**Pull request by blargly**
Data seeder for ref data. This posts data to the new APIs.
Also some changes due to warnings, refactor of some DTOs and Controller end points due to back and forth debates on what the correct response should be for DELETE

### CODE REVIEW

File: src/Common/Common/Data/ReferenceData/AnnualStatisticGroup.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+
+namespace Common.Data.ReferenceData
+{
+    /// <summary>
+    /// Ref Data. A group which contains a collection of
```
Comment (dan-giddins) [inline src/Common/Common/Data/ReferenceData/AnnualStatisticGroup.cs]: Is this comment meant to be like this?

File: src/RefDataLoader/RefDataLoader/Config/ApiSettings.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+using System.Collections.Generic;
+
+namespace Config
+{
+    /// <summary>
+    /// Settings related to the Ref Data API
+    /// </summary>
+    public class ApiSettings
+    {
+        public string Baseuri { get; set; }
```
Comment (dan-giddins) [inline src/RefDataLoader/RefDataLoader/Config/ApiSettings.cs]: is this capitalisation okay?

Comment (beforan) [reply src/RefDataLoader/RefDataLoader/Config/ApiSettings.cs]: `BaseUri` would be better really

File: src/Common/Common/Data/ReferenceData/AnnualStatisticGroup.cs (hunk 1 above)
Comment (blargly) [reply src/Common/Common/Data/ReferenceData/AnnualStatisticGroup.cs]: Amended

File: src/RefDataLoader/RefDataLoader/Config/ApiSettings.cs (hunk 2 above)
Comment (blargly) [reply src/RefDataLoader/RefDataLoader/Config/ApiSettings.cs]: Amended

**Pull request merged**